src/Data/Nat/Binary/Properties.agda

```diff
@@ -654,20 +660,35 @@ private
   { isSemigroup = +-isSemigroup
   }
 
++-commSemigroup : CommutativeSemigroup 0ℓ 0ℓ
```

Comment:
The standard name is `+-commutativeSemigroup`

Reply:
Renamed.